SERVER_DEV_SETUP.md

```diff
@@ -0,0 +1,48 @@
+
+
+# Installing a SORMAS Server for development
+**Note: This guide explains how to set up a SORMAS server on Linux and Windows systems, the latter only being intended for usage on development systems. Please also note that certain parts of the setup script will not be executed on Windows.**
```

Comment:
Is this hint still necessary here, since the guide is exclusively for setting up a development server anyway? What we need in addition here, I think, is another hint that the script used in this approach is crafted for the Docker version.

Reply:
Done